core/common/buffer.hpp

```diff
@@ -222,6 +222,18 @@ namespace kagome::common {
      */
      const std::string_view toString() const;
 
+    /**
+     * @brief return content of bytearray as string
```

Comment:
Comments are identical to `toString()`. Maybe better explain when each method should be used?